x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/Step.java

```diff
@@ -38,6 +38,13 @@ public StepKey getNextStepKey() {
         return nextStepKey;
     }
 
+    /**
+     * Indicates if the step is retryable when it encounters an execution error.
```

Comment:
```suggestion
     * Indicates if the step can be automatically retried when it encounters an execution error.
```

We should be careful in our wording since any step can be retried with the `/_ilm/retry` API, but only certain ones would be (currently) able to be *automatically* retried.